src/main/java/MorseCodePresenter.java

```diff
@@ -0,0 +1,22 @@
+public class MorseCodePresenter
+{
+    private final MorseCodeConverter model;
+    private final MorseCodeFrame view;
+
+    public MorseCodePresenter(MorseCodeConverter model, MorseCodeFrame view)
+    {
+        this.model = model;
+        this.view = view;
+    }
+
+
+    public String toEnglish(String text)
```

Comment:
This method should call a set method in the view. This method should return void.